src/vorta/views/misc_tab.py

```diff
@@ -109,6 +116,41 @@ def populate(self):
 
         self.set_icons()
 
+    def on_setting_update(self, sender, instance: SettingsModel, created=False):
+        """
+        Handle a update of the settings db.
+        Non-PyQt slot for peewee's `playhouse.signals` api.
+        It calls `update_checkbox`.
+
+        Parameters
+        ----------
+        sender : Type[SettingsModel]
+            table sending model
+        instance : SettingsModel
+            The model instance (row) saved.
+        created : bool, optional
+            Whether it was newly created, by default False
+        """
+        if not created and instance.type == 'checkbox':
+            self.update_checkbox(instance.key, instance.value)
+        if instance.key == 'autostart':
+            open_app_at_startup(instance.value)
```

Comment:
Like here ?

Reply:
So basically I've to revert the deletion back to before and keep this ? as It should work the same 🤔  and also pass the test 🤔

Reply:
Subjectively I am unsure whether moving the `open_app_at_startup` call to `misc_tab.py` just to pass the tests is a good idea. Consequently I would revert *TheSpeedX* changes.